source/use-cases/home-resolver.rst

```diff
@@ -14,7 +14,11 @@ In this tutorial we'll look at setting up Unbound as a DNS resolver; Firstly for
 
 
 Setting up Unbound
-------------------
+==================
+
+.. point to installation page
```

Comment:
At the top of each page in the documentation I've written for Routinator, I add an explicit anchor. Sometimes I've add these to specific sections as well, so I can authoritatively refer to them.

For example, if you add this RST notation at the top of the installation page:
```
.. _doc_unbound_installation:
```
You can refer to it anywhere in your document using this notation:
```
:ref:`doc_unbound_installation`
```
This reference will be displayed as the title of the page, e.g. "Installation". Sometimes this is not desired and you want specifc text. Then use for example this notation:
```
:ref:`installation instructions<doc_routinator_installation>`
```
Learn more here: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html